postgis/docker-compose.yaml

```diff
@@ -1,12 +1,24 @@
-version: "3.8"
-
+version: '3.8'
 services:
-  postgres:
-    image: postgis/postgis:13-master
-    container_name: local_postgres
-    volumes:
-      - ./data:/var/lib/postgresql/data
-    env_file:
-      - postgres.env
+  db:
+    image: postgres:14.1-alpine
+    restart: always
```

Comment:
this parameter would be great if `unless-stopped`. Because, you can not stop containers when you should try stop container, only way in this case stop full docker engine or remove container